zipline/utils/pandas_utils.py

```diff
@@ -311,3 +311,58 @@ def days_at_time(days, t, tz, day_offset=0):
         seconds=t.second,
     )
     return (days + delta).tz_localize(tz).tz_convert('UTC')
+
+
+def sliding_apply(df, window_length, f, min_periods=None):
```

Comment:
I left this function here even though it isn't being used anymore because it could still be a useful function, even if  only for testing purposes and such.